lib/galaxy/tool_util/verify/wait.py

```diff
@@ -20,7 +20,6 @@ def wait_on(
     timeout: timeout_type,
     delta: timeout_type = DEFAULT_POLLING_DELTA,
     polling_backoff: timeout_type = DEFAULT_POLLING_BACKOFF,
-    sleep_: Optional[Callable] = None,
 ):
```

Comment:
this is used by the unit tests to "mock" out sleeping and write unit tests for the parameters. I would keep this configurable.

Reply:
Although... maybe add a comment since I'll admit it reads a bit funny 😁